.vib/joomla/cypress/cypress/integration/joomla_spec.js

```diff
@@ -0,0 +1,72 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.contains('Username and password do not match').should('not.exist');
```

Comment:
It could be better to search for the presence of the alert here since the error might happen for other reasons as well. 
In that case, we would also be covering other cases of login failures.